dawn/renderer/components/Editor.js

```diff
@@ -216,7 +216,7 @@ class Editor extends React.Component {
           reject();
         } else {
           count += 1;
-          setTimeout(waitPromise.bind(this, resolve, reject), 1000);
+          setTimeout(waitPromise.bind(this, resolve, reject), 2000);
```

Comment:
Seems overkill. Otherwise, all seconds in Dawn are going to seem like utils.SEC*util.Wait or something. Inevitably there's going to be hardcoding.